### ISSUE TITLE
Figure out a way to not require consumers to have to include internal services in tests

### ISSUE THREAD

**Issue by jkeen** [nypublicradio/ember-hifi#14]
We have an injection helper for acceptance tests, but what about unit tests?

I don't want to require people to have to do this:

```javascript
needs:['service:hifi', 'service:sound-cache', 'service:debug-logger']
```

**Comment by noslouch** [comment]
I was seeing that behavior but then as I integrated to wnyc that behavior went away. Are you seeing that issue?

excuse the brevity -- sent via mobile

On Oct 4, 2016, at 8:05 PM, Jeff Keen <notifications@github.com<mailto:notifications@github.com>> wrote:


We have an injection helper for acceptance tests, but what about unit tests?

I don't want to require people to have to do this:

needs:['service:hifi', 'service:sound-cache', 'service:debug-logger']

�

**Comment by jkeen** [comment]
I think it went away because the version of hifi specified was `^1.0.0` and that might have been before i made `sound-cache` into a service. After I upped it to the latest (after some critical fixes) that problem started appearing again.

**Comment by jkeen** [comment]
This got fixed(ish) with the test helper strategy implemented in `1.2.0`
